packages/ckeditor5-image/tests/manual/imageinline.html

```diff
@@ -0,0 +1,19 @@
+<head>
+	<meta http-equiv="Content-Security-Policy" content="script-src 'self' https://ckeditor.com 'unsafe-inline' 'unsafe-eval'">
+</head>
+
+<script src="https://ckeditor.com/apps/ckfinder/3.5.0/ckfinder.js"></script>
+
+<div id="editor">
+	<h2>Inline image</h2>
+	<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+	<figure class="image ck-widget" contenteditable="false">
+		<img src="https://upload.wikimedia.org/wikipedia/en/a/a9/Example.jpg" alt="Example image">
+		<figcaption class="ck-editor__editable ck-editor__nested-editable ck-hidden ck-placeholder" contenteditable="true" data-placeholder="Enter image caption">
```

Comment:
This should be a clean html so `ck-widget`, `contenteditable="false"`, `ck-editor__editable ck-editor__nested-editable ck-hidden ck-placeholder`, `data-placeholder` should not be there.